app/renderer/js/utils/dnd-util.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const ConfigUtil = require(__dirname + '/config-util.js');
+
+function dnd() {
+	this.initDNDButton();
+	const dnd = !ConfigUtil.getConfigItem('dnd', false);
+	const settingList = ['showNotification', 'silent', 'flashTaskbarOnMessage'];
+	let oldSettings;
```

Comment:
Remove this from here and initialize variable inside the `if`/`else` statement.

Reply:
I have to initialize it outside if/else. Due to scoping issue. It is inaccessible outside the block if I initialize it inside the conditional block.

Reply:
Ignore this, i was think of change of variable and stuff :)